src/JWT.php

```diff
@@ -94,7 +94,8 @@ class JWT
      */
     public static function decode(
         string $jwt,
-        $keyOrKeyArray
+        $keyOrKeyArray,
+        &$headers = null
```

Comment:
Let's make this typehinted
```suggestion
        stdClass &$headers = null
```